docs/index.rst

```diff
@@ -37,9 +25,8 @@ Usage
 
 Initialize an database::
 
-    cnx-db init [-h host] [-p port] -d dbname -U user
-
-.. todo:: Deprecate ``cnx-archive-initdb`` in favor of ``cnx-db init ...``.
+    export DB_URL=postgresql:///repository
+    cnx-db init
 
 .. todo:: This may become part of ``dbmigrator init`` or ``dbmigrator migrate``
```

Comment:
Hmm, a standard for dbmigrator could be to look for a migration named "init" in the context[s] and run th[at|em] when initing an empty db? (I realize this comment is out-of-scope for PR)

Reply:
Create an enhancement issue in db-migrator?